### PR TITLE
Integrate multi arch support

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: docker/Dockerfile.linux.arm32v6 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM arm32v6/alpine:3.6
+EXPOSE 3000
+
+ENV GODEBUG netdns=go
+
+RUN apk add -U --no-cache ca-certificates
+
+ADD release/linux/amd64/drone-amazon-secrets /bin/
```
Comment (bradrydzewski) [inline docker/Dockerfile.linux.arm32v6]: s/amd64/arm

File: docker/Dockerfile.linux.arm32v7 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM arm32v7/alpine:3.6
+EXPOSE 3000
+
+ENV GODEBUG netdns=go
+
+RUN apk add -U --no-cache ca-certificates
+
+ADD release/linux/amd64/drone-amazon-secrets /bin/
+ENTRYPOINT ["/bin/drone-amazon-secrets"]
```
Comment (bradrydzewski) [inline docker/Dockerfile.linux.arm32v7]: s/amd64/arm

File: docker/Dockerfile.linux.arm64v8 [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FROM arm64v8/alpine:3.6
+EXPOSE 3000
+
+ENV GODEBUG netdns=go
+
+RUN apk add -U --no-cache ca-certificates
+
+ADD release/linux/amd64/drone-amazon-secrets /bin/
```
Comment (bradrydzewski) [inline docker/Dockerfile.linux.arm64v8]: s/amd64/arm64

**Comment by bradrydzewski** [comment]
before we finalize an approach, I'm interested to see what @metalmatze thinks about how we can use jsonnet to clean this up:
https://github.com/drone/drone-amazon-secrets/blob/16aa91abd33cb52ea4f8c5edf6c061cc241051c1/.drone.yml

**Comment by metalmatze** [comment]
It's probably easiest to figure out the common things in the document and then create helpers with sane defaults that can be extended.

**Comment by tboerger** [comment]
If I just fix the Dockerfile binary naming should we just merge it without the .drone.yml for now and add the .drone.yml later on?

**Comment by tboerger** [comment]
@bradrydzewski please check if this could be a proper standard format for you which supports multi arch automated deployments :)

**Pull request closed without merging**